app/services/activity-service.js

```diff
@@ -58,10 +89,11 @@ export default class activityService extends Service {
 
     // publishPreviewActivityType.
     const requestPublishPreviewActivityType = await this.store.findRecord('activity-type', CONFIG.ACTIVITY_TYPES.drukproeven.id);
+    const requestActivityStatus = await this.store.findRecord('activity-status', CONFIG.ACTIVITY_STATUSSES.open.id);
```

Comment:
Ik zou hier misschien nog vermelden welke status het is in de variabele.. op zich zie je het wel in de `CONFIG.ACTIVITY_STATUSSES.open.id` maar het lijkt me sneller als je het in de variabele kunt lezen.. anders moet je nog verder kijken over welke status het gaat.